### PR TITLE
Dev pull to main

### PR DESCRIPTION
**Pull request by cwverica**
It's all a bit ugly right now, but it's starting to come together.
Still plenty of work ahead.
With this commit to be merged:
-Front loader works properly, loads ALL recipe_ingredients 
before logic error was only loading one recipe_ingredient per recipe
-Search logic now works
-Displays 10 results on the page. Will display the next 10 on refresh. (need to work on pagination)

Let's pull this to the main now, and I'll keep working on navigation.

### CODE REVIEW

File: front_loader.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    try:
+        db.session.add(new_recipe)
+        db.session.commit()
+        print("drink added")
+    except Exception as err:
+        print('error', err)
+
+
+def add_recipe_ingredients_relationships(drink):
+    for i in range(1,16):
```
Comment (montrealist) [inline front_loader.py]: "Magic number" 16 - please avoid magic numbers like this. They are confusing and make it difficult to reason about your code.

File: templates/display_results.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
```
Comment (montrealist) [inline templates/display_results.html]: Is this template used to display search results? If so might make sense to call it `search_results.html`

File: app_logic.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    compile containing set
+
+for id in set:
+    pull full recipe
+    create dict for recipe
+    include item how many ingredients match
+'''
+
+
+def get_recipe_ids_with_ingredient(ingredient):
```
Comment (montrealist) [inline app_logic.py]: Why is there a function with an identical name in another file? Is this one used? If not delete it.

https://github.com/cwverica/quixr/pull/1/files#diff-568470d013cd12e4f388206520da39ab9a4e4c3c6b95846cbc281abc1ba3c959R235

Comment (cwverica) [reply app_logic.py]: This whole file is probably going to get deleted. I was using it before I decided to front-load the app, and it just kinda became a scratch-pad.

File: templates/display_results.html (hunk 2 above)
Comment (cwverica) [reply templates/display_results.html]: Will do

File: front_loader.py (hunk 1 above)
Comment (cwverica) [reply front_loader.py]: I think maybe I should just leave a comment? The reasoning here is that the API leaves room for 15 ingredients, and 15 corresponding measurements.  It rarely uses more than 10.

Or I guess I could just change the loop to be a while, and set it to break when it encounters a None

**Review by montrealist — APPROVED** [review]

**Pull request merged**